test/e2e/handler/webhook_test.go

```diff
@@ -18,12 +18,10 @@ var _ = Describe("Mutating Admission Webhook", func() {
 			// Make sure test policy is not there so
 			// we exercise CREATE event
 			resetDesiredStateForNodes()
-			updateDesiredState(linuxBrUp(bridge1))
+			updateDesiredStateAndWait(linuxBrUp(bridge1))
 		})
 		AfterEach(func() {
-			waitForAvailableTestPolicy()
```

Comment:
Don't we need the first waitForAvailableTestPolicy() ?

Reply:
We have move the wait to the proper places I think we don't need this anymore here right ? this test know just check that the label is there we don't care if it's after Status is good (for webhook label is set after we finish applying the manifests but wait is good to be sure we are at finished status).